conmon/conmon.c

```diff
@@ -1235,11 +1235,12 @@ int main(int argc, char *argv[])
 		exit(0);
 	}
 
+	if (opt_cid == NULL)
+		fprintf(stderr, "Container ID not provided. Use --cid");
```

Comment:
You need to `exit(EXIT_FAILURE);` in this block too.

Reply:
Fixed.